src/broker/cas_function.c

```diff
@@ -90,6 +91,8 @@ static void update_error_query_count (T_APPL_SERVER_INFO * as_info_p, const T_ER
 
 static const char *tran_type_str[] = { "COMMIT", "ROLLBACK" };
 
+static void ddl_node_type_find_and_set (T_SRV_HANDLE * srv_handle);
```

Comment:
static void cub_ddl_log_set_node_type (T_SRV_HANDLE * srv_handle);

Reply:
Thank you, i will change code as your recommend.